client/src/pages/index.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import { css } from 'react-emotion';
+
+const topnav=css`
+    background: #000099;
+    overflow: hidden;
+    height:68px;
+    
+`
+
+const buttonposition=css`
+    float:right;
+    margin:21px 35px 35px;
+`
+const formborder=css`
+    border-radius: 5px;
+    background-color: #f2f2f2;
+    padding: 20px;
+`
+class Logincontrol extends Component {
+  constructor(props) {
+    super(props);
+    this.handleLoginClick = this.handleLoginClick.bind(this);
+    this.handleLogoutClick = this.handleLogoutClick.bind(this);
```

Comment:
Arrow functions can be used in place of binding like this.
`exampleMethod = ( ) => {
  //this is bound to the class
}`